submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/cluster/web/ClusterExecutor.kt

```diff
@@ -1,17 +1,20 @@
 package ac.uk.ebi.biostd.cluster.web
 
 import ac.uk.ebi.biostd.common.properties.ApplicationProperties
+import ac.uk.ebi.biostd.common.properties.Cluster
+import ac.uk.ebi.biostd.submission.config.GeneralConfig
 import uk.ac.ebi.biostd.client.cluster.api.LsfClusterClient
+import uk.ac.ebi.biostd.client.cluster.api.SlurmClusterClient
 import uk.ac.ebi.biostd.client.cluster.model.Job
 import uk.ac.ebi.biostd.client.cluster.model.JobSpec
 
-class ClusterExecutor private constructor(val lsfClusterClient: LsfClusterClient) {
+class ClusterExecutor private constructor(
+    val lsfClusterClient: LsfClusterClient,
```

Comment:
And finally, we could use the interface and pass the correct implementation based on the cluster mode rather than checking it here

Reply:
I create this class with the main purpose of actually supporting both cluster and being able to test both, I think will this class will be useless once codon disapeer completly but I do think it will be usefull during transition period.